Wrappers/Python/test/test_reconstructors.py

```diff
@@ -604,10 +596,9 @@ def test_results_inplace_filtering(self):
         self.assertGreater(diff,0.8)
 
 
-class Test_FBP_results(unittest.TestCase):
-
+@unittest.skipUnless(has_tigre and has_nvidia and has_ipp, "TIGRE or IPP not installed")
+class Test_FBP_tigre_ipp(unittest.TestCase):
```

Comment:
These aren't just tigre tests, under this class we have some tigre and some astra, so potentially this needs to be split in to two classes?

Reply:
moved to #1758